GalaxyZooTouchTable/GalaxyZooTouchTable/GalaxyZooTouchTable.csproj

```diff
@@ -386,6 +386,7 @@
     <None Include="App.Release.config">
       <DependentUpon>App.config</DependentUpon>
     </None>
+    <Resource Include="Fonts\Garamond.ttf" />
```

Comment:
Looks good. Would it make sense for us to do a quick pass on the other fonts and make sure they all show up on the touch table as well? Or is the issue that the text doesn't show up at all (as opposed to showing up in a default font)?

Reply:
Other fonts (Karla and Poppins) are both showing up, mainly due to the fact their .ttf files are packed straight into the app. Garamond is the only font that I didn't include a .tff file along with the app. I did this because Garamond is a common font that should be on every system. However, it isn't showing up on the touch table itself, whereas it is showing up on other machines. In the case of the touch table, it currently falls back on a default font.

Reply:
Sounds like we now have all the fonts packaged up with the app, then. Works for me!